### ISSUE TITLE
UWP? Really?

### ISSUE THREAD

**Issue by ansorg** [UWP-Open-Source-Community/Documentation#6]
Centennial != UWP

**Comment by manuelelucchi** [comment]
Is a bit difficult. Even MS doesn't know how to call centennial apps. They are Win32/.NET code on a UWP container, with UWP API used. We generally call them UWP

**Comment by ansorg** [comment]
Agree, it's confusing. Calling them 'packaged desktop apps' should be fine. Until they are actually re-developed fully as UWP.

**Comment by manuelelucchi** [comment]
I could agree with you, but there are some apps with uwp interface and win32 core. Many others with win32 interface and uwp core. How can we call them? It's difficoult
